homeassistant/components/plugwise/climate.py

```diff
@@ -117,6 +110,21 @@ def hvac_mode(self) -> HVACMode:
             return HVACMode.HEAT
         return HVACMode(mode)
 
+    @property
+    def hvac_modes(self) -> list[HVACMode]:
+        """Return the list of available HVACModes."""
+        hvac_modes: list[HVACMode] = [HVACMode.HEAT]
```

Comment:
That is already inferred.

```suggestion
        hvac_modes = [HVACMode.HEAT]
```